docs/Installing.asciidoc

```diff
@@ -99,13 +135,66 @@ You can do that in +/etc/openqa/openqa.ini+:
 httpsonly = 0
 --------------------------------------------------------------------------------
 
+=== Database
+
+Since verion _4.5.1512500474.437cc1c7_ of openQA, PostgreSQL is used as the
+default database.
+
+It is possible to configure other databases.
+To configure access to the chosen database in openQA,
+edit +/etc/openqa/database.ini+ and change the settings in the +[production]+
+section.
+
+The +dsn+ value format technically depends on the database type and is
+documented for PostgreSQL at
+https://metacpan.org/pod/DBD::Pg#DBI-Class-Methods[DBD::Pg]
+
+====== Example for connecting to local PostgreSQL database
+
+[source,ini]
+--------------------------------------------------------------------------------
+[production]
+dsn = dbi:Pg:dbname=openqa
+--------------------------------------------------------------------------------
+
+====== Example for connecting to remote PostgreSQL database
+
+[source,ini]
+--------------------------------------------------------------------------------
+[production]
+dsn = dbi:Pg:dbname=openqa;host=db.example.org
+user = openqa
+password = somepassword
+--------------------------------------------------------------------------------
+
+====== Older versions of openQA
+
+Older versions of openQA, like the one distributed on Leap 42.3, had support for
+SQLite, but the support was dropped since openQA version
+_4.5.1512500474.437cc1c7_ and now the default database is PostgreSQL.
+
+If you are installing an older version than _4.5.1512500474.437cc1c7_ of
+openQA, SQLite is the default database, and is ready to use, but it is
+recommended to migrate to PostgreSQL.
+
+If you have an openQA version older than _4.5.1512500474.437cc1c7_ and you
+want to use PostgreSQL, you need to create a database (openqa) and a dedicated
+user account (geekotest) with full access to it.
+
+[source,sh]
+-------------------------------------------------------------------------------
+# Assuming that you already have installed PostgreSQL in your system
+su - postgres -c 'createuser -D geekotest'
+su - postgres -c 'createdb -O geekotest openqa'
+-------------------------------------------------------------------------------
+
```

Comment:
When https://github.com/os-autoinst/openQA/pull/1530 gets merged, crosslink here "Migrating from existing SQLite instance to PostgreSQL.